### PR TITLE
set up a task for create a report by console

### PR DESCRIPTION
**Pull request by raulchoque-eng**

### CODE REVIEW

**Review by JosueRodriguezGarcia — APPROVED** [review]

**Review by LimbertVargas — DISMISSED** [review]
it's ok for me... good job Raul

File: build.gradle [hunk 1]
```diff
@@ -13,6 +13,8 @@ plugins {
     id 'findbugs'
     id 'pmd'
     id 'checkstyle'
+    id "com.github.spacialcircumstances.gradle-cucumber-reporting" version "0.1.8"
+
```
Comment (LimbertVargas) [inline build.gradle]: please delete this space.

Comment (raulchoque-eng) [reply build.gradle]: thanks, it mistake was fixed

**Review by LimbertVargas — APPROVED** [review]
good job

**Pull request merged**